### ISSUE TITLE
Show mod list for servers in mainmenu

### ISSUE THREAD

**Issue by Lejo1** [minetest/minetest#8782]
## General

I already created this PR #8691 to pass the clients_list and the mods to the lua mainmenu.
But I'm not sure how to list it there.
I already created one possible solution. It's my [player_mod_list](https://github.com/Lejo1/minetest/tree/player_mod_list) branch
I just use two dropdowns.

![Bildschirmfoto von 2019-08-12 16-13-19](https://user-images.githubusercontent.com/18502734/62872711-57ba3380-bd1e-11e9-8444-e7e86dfa7eab.png)

Ideas for different designs are welcome!

**Comment by rubenwardy** [comment]
I'd sort both lists the append to the description text box

**Comment by paramat** [comment]
The dropdowns in the screenshot are too narrow and cramped, mod names will often be longer, especially when the game window is set to a small size.

**Comment by Lejo1** [comment]
I also noticed that.
What about only listing the players? The most gamers don’t need any mod listing. In Addition to this not every server sends the mods to the serverlist

**Comment by Lejo1** [comment]
The other idea is to make a mod+player search just like this:
https://forum.minetest.net/viewtopic.php?f=14&t=22313

**Comment by Enrikoo** [comment]
Always when I compile it, it doesnt list anything where I can see players and mods, how is it just possible for you?

**Comment by Lejo1** [comment]
If you compile my player_mod_list branch and select a server you should see these Dropdowns.

**Comment by Enrikoo** [comment]
> If you compile my player_mod_list branch and select a server you should see these Dropdowns.

Wait where is your branch? And I dont know what to do with it. I compiled your minetest pr but it didnt list anything like this in the image.

**Comment by Lejo1** [comment]
Just compile [this](https://github.com/Lejo1/minetest/archive/player_mod_list.zip) Minetest

**Comment by Enrikoo** [comment]
> Just compile [this](https://github.com/Lejo1/minetest/archive/player_mod_list.zip) Minetest

It worked, thx for the tip.

**Comment by Lejo1** [comment]
> I'd sort both lists the append to the description text box

That's the other idea:
![playerlist](https://user-images.githubusercontent.com/18502734/74606312-bf877780-50cf-11ea-945e-8fa5c56659d9.png)
What do you think is the best way?

**Comment by paramat** [comment]
Appending to the description box seems much better.

**Comment by LoneWolfHT** [comment]
Needs to be more visually separate from the description text. Possibly with a little more space in between/a different background

**Comment by paramat** [comment]
I agree, empty lines between sections would be good.

**Comment by rubenwardy** [comment]
Remind me, are there any colorable wrapping containers?  Hypertext maybe?

**Comment by Lejo1** [comment]
> Needs to be more visually separate from the description text. Possibly with a little more space in between/a different background

![mainmenu](https://user-images.githubusercontent.com/18502734/75446870-84573500-5968-11ea-8be8-2c4e913041e5.png)
Description-Text will become much longer in most cases.

Anyway still requires #8691

**Comment by EvolutionPixG** [comment]
> 
> 
> > Needs to be more visually separate from the description text. Possibly with a little more space in between/a different background
> 
> ![mainmenu](https://user-images.githubusercontent.com/18502734/75446870-84573500-5968-11ea-8be8-2c4e913041e5.png)
> Description-Text will become much longer in most cases.
> 
> Anyway still requires #8691

The list of players and mods I think should be separated by a comma.

**Comment by nerzhul** [comment]
i suggest to have a popup to show more infos, with a button, it's far more better :)

**Comment by Lejo1** [comment]
> i suggest to have a popup to show more infos, with a button, it's far more better :)

Yes, I agree: https://github.com/Lejo1/minetest/tree/player_list_popup
![serverlistb](https://user-images.githubusercontent.com/18502734/81094488-dd60bd80-8f03-11ea-8a0d-ec1c32d26cd8.png)
![serverlistp](https://user-images.githubusercontent.com/18502734/81094491-dfc31780-8f03-11ea-8c07-7fa017089d78.png)

**Comment by appgurueu** [comment]
The client list part of this has been resolved by #15425, only the mod list part remains.
